dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/FileAssetAPIImpl.java

```diff
@@ -58,6 +58,7 @@
  */
 public class FileAssetAPIImpl implements FileAssetAPI {
 
+    private final static String DEFAULT_RELATIVE_ASSET_PATH = "/assets";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)